test_scripts/API/GetWayPoints/common_get_way_points.lua

```diff
@@ -0,0 +1,273 @@
+---------------------------------------------------------------------------------------------------
+-- GetWayPoints common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.defaultProtocolVersion = 2
+config.ValidateSchema = false
+
+--[[ Required Shared libraries ]]
+local mobile_session = require("mobile_session")
+local json = require("modules/json")
+
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+local commonSteps = require("user_modules/shared_testcases/commonSteps")
+local commonTestCases = require("user_modules/shared_testcases/commonTestCases")
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local hmi_values = require("user_modules/hmi_values")
+
+--[[ Local Variables ]]
+local ptu_table = {}
+local hmiAppIds = {}
+
+local common_get_way_points = {}
```

Comment:
@LitvinenkoIra I propose to use module name variable in camelCase:
`local commonWayPoints` 
or
`local commonWP`

Reply:
@LitvinenkoIra +1